### PR TITLE
use latest Rubies in CI

### PR DESCRIPTION
**Pull request by takkanm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,19 +10,19 @@ after_success:
...
         - bundle exec danger
-    - rvm: 2.4.0
-    - rvm: 2.3.3
-    - rvm: 2.2.6
+    - rvm: 2.4.1
+    - rvm: 2.3.4
+    - rvm: 2.2.7
     - rvm: ruby-head
-    - rvm: jruby-9.1.6.0
+    - rvm: jruby-9.1.9.0
```
Comment (koic) [inline .travis.yml]: JRuby 9.1.10.0 was released.
http://jruby.org/2017/05/25/jruby-9-1-10-0.html

Comment (LeFnord) [inline .travis.yml]: good spot …

**Comment by takkanm** [comment]
I updated.

**Comment by LeFnord** [comment]
thanks @takkanm

**Pull request merged**